### ISSUE TITLE
package breaks meteor app on IE9

### ISSUE THREAD

**Issue by Twisterking** [aldeed/meteor-schema-index#8]
Hi there!

So unfortunately your package is breaking my Meteor app for IE9 (many of my customer still use IE9 - don't ask! big companies preventing updates and all that ...).

Problem is here: [link to file](https://github.com/aldeed/meteor-schema-index/blob/master/lib/indexing.js#L45)
This file is included on the client as well and you are using `const` right there. IE9 can't handle `const` --> syntax error.
Any chance on fixing this?

best, P

**Comment by Twisterking** [comment]
I think what you are missing is:

```
Package.onUse(function (api) {
  api.use('ecmascript');
});
```

see [here](https://github.com/meteor/meteor/blob/master/packages/ecmascript/README.md#usage).

**Comment by lassombra** [comment]
Both #7 and #9 provide solutions to this.

**Comment by chrisbutler** [comment]
hey @aldeed, this is a big issue... any timeline for releasing the fix?

**Comment by aldeed** [comment]
Released the fix with 1.1.1. Thanks
